openfl/interface/workspace.py

```diff
@@ -504,10 +510,15 @@ def open_pipe(command: str):
 
     grainized_ws_dockerfile = SITEPACKS / 'openfl-gramine' / 'Dockerfile.graminized.workspace'
 
+    if tag is None:
+        tag = workspace_name
+    else:
+        tag = tag.format(workspace_name=workspace_name)
```

Comment:
```suggestion

```
`tag` is already a string

Reply:
this is meant to allow the user to add the workspace name to the tag. for example, if the name of the workspace is `test_workspace` then `prefix-{workspace_name}-suffix` will translate into `prefix-test-workspace-suffix`

Reply:
i needed a way to override the name of the tag/workspace when working on automated image build and execution. i agree that having separate tag and workspace names could be too confusing, so perhaps the following may be more appropriate
1. tag and workspace name will be the same (current behavior in `graminize` and `dockerize`)
2. instead of tag name (`-t`), add an argument to override workspace name: `-w workspace_name`
3. if provided, the workspace name will be overridden and used as tag for the image
4. if not, apply current behavior

Reply:
As I said above, `workspace_name` is not a parameter to the `graminize` command, it is a part of the context as the command must be called from some workspace (it has a name already)

Reply:
note that i recommitted a simpler version in which only the workspace name can be overridden and the tag is always the same